### PR TITLE
RN updated for ksp 1.2

### PR DESCRIPTION
**Pull request by raidernick**
-Update all mods for ksp 1.2
-Removed all unneeded dependencies, I now only use bahaSP and sometimes
texture replacer for my mods. Firespitter and animated dec are no longer
needed.
- Updated all craft files so they can be installed by default again

### CODE REVIEW

**Comment by raidernick** [comment]
@politas this may fail to validate as smokescreen and texture replacer aren't officially updated yet. They are both not REQUIRED for my mods to work, only for pre-launch smoke effects and some reflections on probes. The mod will function perfectly fine without them.

**Comment by politas** [comment]
If they aren't required, you could move them into "recommends" rather than "depends". Otherwise, your mods will disappear from the CKAN list when this is merged.

**Comment by raidernick** [comment]
@politas I meant the mods will "work" without the plugins, but for the probes it is essential that they are reflective, like the sputnik, it is not meant to be matte. I could change it, but I don't want to give the option for the users to disable installing it. Smokescreen I will consider. Can you give me a few days to think about this before merging it. I'll ping you again on here when/if I decide to make changes and have it merged.

**Comment by politas** [comment]
Can't see a reason not to move TextureReplacer to another maintainer if the original author is MIA

**Comment by raidernick** [comment]
@politas I made the changes to TR and changed smokescreen to recommended. Let me know if that looks good. If all is acceptable and this will allow my mods to show up for ksp 1.2, this is good to merge.

File: NetKAN/RN-USProbesPack.netkan [hunk 1]
```diff
@@ -26,8 +26,10 @@
     "depends": [
         { "name": "BDAnimationModules" },
         { "name": "TextureReplacer" },
-        { "name": "FirespitterCore" },
-		{ "name": "ModuleManager" },
-        { "name": "SmokeScreen" }
+		{ "name": "ModuleManager" }
+    ]
```
Comment (Olympic1) [inline NetKAN/RN-USProbesPack.netkan]: Forgot a comma here

**Comment by raidernick** [comment]
@Olympic1 added it

**Comment by Olympic1** [comment]
Thx. Jenkins loves it, everything installs fine locally, so merging with my thanks

**Pull request merged**